deploy/chart/values.yaml

```diff
@@ -58,6 +58,13 @@ nodePathMap:
     paths:
       - /opt/local-path-provisioner
 
+# By default only ReadWriteOnce mount mode is available as host path is usually exist on
+# the node where it was created. In some setups path can represent shared filesystem
+# as NFS/Lustre/BeeGFS, etc., so the same path is available on multiple nodes of the cluster.
+# "allowMultiNode: true" allow to create volumes with other access modes:
+# ReadOnlyMany and ReadWriteMany
+allowMultiNode: false
```

Comment:
I agree that this might cause confusion. I would rather just add `sharedPath` key which is active when `allowMultiNode: true` to avoid this confusion. There might be a use case when a shared filesystem mounted on different paths on every node, but I would assume it is a rare setup.